src/components/cards/PoweredByCard.tsx

```diff
@@ -54,6 +54,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
     flexDirection: 'column'
   },
   poweredByContainer: {
+    marginHorizontal: theme.rem(0.25),
```

Comment:
Though the margins were increased, they're still less than the rest of the app and it makes this stand out in a bad way. I said this before in an earlier PR, I don't think reducing margins is appropriate to "de-emphasize" a card. To me it doesn't convey any sort of de-emphasis, but looks messy and inconsistent when the padding is different. Madison also brought this up without bringing it up to her and expressed the same views.

I would really prefer just using the same UI4 components we already have for this component instead of a custom one-off implementation. If we really want something different, then we should design a different component instead. 

Honestly, I don't think there is even a need to de-emphasize this at all.

Reply:
So let's take up the discussion on whether or not it "needs de-emphasis" as a separate task/sidebar. This commit is to address the inconsistent padding for that button so that the top/bottom match left/right.